src/db/commit.fbs

```diff
@@ -0,0 +1,32 @@
+namespace commit;
+
+table LocalMeta {
```

Comment:
can we put comments in the definition files? if so I think it would be helpful. f